File: 프로그래머스/unrated/181939. 더 크게 합치기/더 크게 합치기.js
function solution(a, b) {
    a += '';
    b += '';
    
    if (a + b > b + a) {
        return (a + b) * 1;
    } else if (a + b < b + a) {
        return (b + a) * 1;
    } else {
        return (a + b) * 1;
    }
}

File: 프로그래머스/unrated/181939. 더 크게 합치기/더 크게 합치기.java
class Solution {
    public int solution(int a, int b) {
        int aPlusB = Integer.parseInt(String.valueOf(a) + String.valueOf(b));
        int bPlusA = Integer.parseInt(String.valueOf(b) + String.valueOf(a));
        
        return aPlusB >= bPlusA ? aPlusB : bPlusA;
    }
}